torchci/components/JobLinks.tsx

```diff
@@ -49,7 +49,9 @@ export default function JobLinks({
         )}&jobName=${encodeURIComponent(
           job.jobName as string
         )}&failureCaptures=${encodeURIComponent(
-          JSON.stringify(job.failureCaptures)
+          job.failureCaptures.length == 1
```

Comment:
Is this simpler to just keep this part (I assume that people will just copy the full url anyway)?  We will just need to have a text field `<TextField label="Failure Captures" defaultValue={failureCaptures} />` then put the string input into an array behind the scene?

Reply:
I'm not sure I understand what you're getting at here.  Like the failure page should always have failureCaptures be string[] and just show the one string if it's length one?

Reply:
I mean the PR changes the `api/failure.ts` API to accept either a string or a list of string via the use of `formatFailureCaptures` function.  IMO, it seems easier to just always accept a list of string to match what the log classifier has (it means to capture multiple failures).  The caller of `api/failure.ts` from `torchci/pages/failure.tsx` can put the failure into a list before passing it to the API?

Reply:
I see, done